intermediate_source/reinforcement_q_learning.py

```diff
@@ -386,11 +314,9 @@ def plot_durations():
 # :math:`V(s_{t+1}) = \max_a Q(s_{t+1}, a)`, and combines them into our
 # loss. By definition we set :math:`V(s) = 0` if :math:`s` is a terminal
 # state. We also use a target network to compute :math:`V(s_{t+1})` for
-# added stability. The target network has its weights kept frozen most of
-# the time, but is updated with the policy network's weights every so often.
-# This is usually a set number of steps but we shall use episodes for
-# simplicity.
-#
+# added stability. The target network is updated at every step with a 
+# `soft update <https://arxiv.org/pdf/1509.02971.pdf>`__ controlled by 
+# the hyperparameter ``TAU``, which was previously defined. 
```

Comment:
@SiftingSands Previously there was always a new line ending the comment blocks -- I wonder if that's the reason why the html rendering is broken. Would you mind adding the empty comment line (`#`) back?

Reply:
Haha it looks like we typed at the same time and found the same culprit. I'll give that a shot

Reply:
I still can't see it... :/